### DISCUSSION TITLE
Filtering with group fields

### DISCUSSION THREAD

**Discussion by gamellis** [payloadcms/payload#1686] [Feature Requests & Ideas]
Hello! Any plans to allow fields within a group to be filtered? Thanks for all your fantastic work! I keep bragging about how fast we're moving with payload ❤️ 

<img width="600" alt="Screen Shot 2022-12-16 at 9 45 41 AM" src="https://user-images.githubusercontent.com/5551288/208158014-7f91115a-32b5-4385-8c45-32f97634265d.png">
Fields within a group are not listed.

Thanks again!

**Comment by jmikrut** [comment]
Hey @gamellis — welcome!!

Pumped to hear about your speed while working with Payload.

This should be absolutely doable. I will mark it as Roadmap priority 2. In the interim, it could be a possible PR coming from the community too. I don't think it would be too hard. You can already query on fields nested in a group just by using dot notation - i.e. `where[myGroup.subField][equals]=test` so we would just have to surface the nested fields and then handle formatting the field "path". Not too bad.

**Comment by andershermansen** [comment]
This is already fixed in #6647 and available in latest 3.0 beta.

**Reply by gamellis** [reply]
Amazing! Thanks @andershermansen

**Comment by DanRibbens** [comment]
This is being added to v2 also.

https://github.com/payloadcms/payload/pull/10421

Thanks @JessChowdhury!
